docs/source/Tutorial.md

```diff
@@ -1134,6 +1399,14 @@ before:
   }
 ~~~
 </div>
+<div class="language-c">
+~~~{.c}
+  #include "monster_reader.h"
+
+  #define ns(x) FLATBUFFERS_WRAP_NAMESPACE(MyGame_Sample, x) // Specified in the schema.
+  #define nsc(x) FLATBUFFERS_WRAP_NAMESPACE(flatbuffers, x)
```

Comment:
The generated code is already prefixed. The ns() style is merely a convenience that user can avoid, espeically when namespaces are absent or short. In this, all identifiers are prefixed MyGame_Sample_, which is too long to be readable.